vsphere/resource_vsphere_datacenter_test.go

```diff
@@ -0,0 +1,124 @@
+package vsphere
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/vmware/govmomi"
+	"github.com/vmware/govmomi/find"
+	"golang.org/x/net/context"
+)
+
+// Create a datacenter on the root folder
+func TestAccVSphereDatacenter_createOnRootFolder(t *testing.T) {
+	const resourceName = "vsphere_datacenter.testDC"
+
+	const testAccCheckVSphereDatacenterConfig = `
```

Comment:
Can we move these constants to package scope? We don't really have many constants that have function scope.